Core/FileStore.swift

```diff
@@ -36,6 +36,21 @@ public class FileStore {
         }
     }
     
+    func removeData(forFile file: String) -> Bool {
+        var path = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: groupIdentifier)
+        path = path!.appendingPathComponent(file)
+        guard let path = path else { return false }
+        guard FileManager.default.fileExists(atPath: path.absoluteString) else { return true }
```

Comment:
Please call `path` a `fileURL` and use `fileURL.path` instead of `absoluteString` as the latter doesn't work with `removeItem(at:)` api.